modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -20,6 +20,7 @@ In order to use horizontal pod autoscalers, your cluster administrator must have
 You can use the `oc describe PodMetrics <pod-name>` command to determine if metrics are configured. If metrics are
 configured, the output appears similar to the following, with `Cpu` and `Memory` displayed under `Usage`.
 
+[source,terminal]
 ----
 $ oc describe PodMetrics openshift-kube-scheduler-ip-10-0-135-131.ec2.internal
 ----
```

Comment:
Should the yaml file be preceded by ".Example output"? If not, I think it's clear as is.